ui/perfherder/graphs/GraphsViewControls.jsx

```diff
@@ -80,6 +82,13 @@ export default class GraphsViewControls extends React.Component {
               Add test data
             </Button>
           </Col>
+          {!hasNoData && (
+            <Col sm="auto" className="p-2">
+              <Button color="darker-info" onClick={toggleTableView}>
+                {showTable ? 'Graphs View' : 'Table View'}
+              </Button>
+            </Col>
+          )}
```

Comment:
You can move this to line 101, including it in the second condition of the ternary statement that begins on line 94 (if `hasNoData` is true, do this, otherwise do that (starting on line 101).

Reply:
Hmm I have changed the button position per @alexandru-io's request and applying this suggestion now would alter its ordering.

Reply:
If it's no longer applicable, that's fine :)